### PR TITLE
added notices page

### PR DESCRIPTION
**Pull request by Akalanka47000**
-noticeboard image needs replacement

Screenshot_1
Screenshot_2
Screenshot_3
Screenshot_4
Screenshot_5
Screenshot_6
Screenshot_7

https://user-images.githubusercontent.com/73662613/124648355-caa19a00-deb4-11eb-89ad-238d7d554d82.mp4

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
Next time If you create a pull request please be kind to assign reviewers,project and assign what is the issue of that PR.

**Comment by Akalanka47000** [comment]
> @Akalanka47000 Next time If you create a pull request please be kind to assign reviewers,project and assign what is the issue of that PR.

Will do from next time

File: components/carousel/notice/NoticeCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <div
+      data-aos="fade-up"
+      className="flex px-5 pb-14 md:pb-24 pt-5 md:pt-10 items-center justify-center flex-col"
+    >
+      <Carousel
+        className="transition ease-in duration-200"
+        showThumbs={false}
+        showArrows={false}
+        showStatus={false}
+        showIndicators={false}
```
Comment (Shehanka) [inline components/carousel/notice/NoticeCarousel.tsx:37]: make these false props are optional. and set default values are false if they are not set as true

**Review by Shehanka — CHANGES_REQUESTED** [review]

File: components/carousel/notice/NoticeCarousel.tsx (hunk 1 above)
Comment (Akalanka47000) [reply components/carousel/notice/NoticeCarousel.tsx:37]: it's from a package, the default values were true

Comment (Shehanka) [reply components/carousel/notice/NoticeCarousel.tsx:37]: > it's from a package, the default values were true

I want to meet him 😾

**Review by sithumonline — APPROVED** [review]
LGTM

**Review by Shehanka — APPROVED** [review]

File: components/carousel/notice/NoticeCarousel.tsx (hunk 1 above)
Comment (Akalanka47000) [reply components/carousel/notice/NoticeCarousel.tsx:37]: 😂😂😂

**Pull request merged**